pkgs/applications/video/kodi/default.nix

```diff
@@ -72,9 +69,9 @@ let
 
   ffmpeg = kodiDependency rec {
     name    = "FFmpeg";
-    version = "3.1.11";
-    rev     = "${version}-${rel}-17.5"; # TODO: change 17.5 back to ${kodiVersion}
-    sha256  = "0nc4sb6v1g3l11v9h5l9n44a8r40186rcbp2xg5c7vg6wcpjid13";
+    version = "4.0.3";
+    rev     = "${version}-${rel}-RC5";
```

Comment:
Is this still correct?

Reply:
Latest from upstream, so yes

Reply:
Yeah, seems like they don't bump all their dependencies for a release…